docs/api/headers.md

```diff
@@ -29,11 +29,11 @@ console.log(authHeader); // "Bearer 123456789"
 
 ## Referer
 
-The `Referer` header is a special header that is automatically set by `jstz` when it makes a request. The value of the `Referer` header is the `tz4` address of the smart function (or account) that made the request.
+The `Referer` header is a special header that is automatically set by `jstz` when it makes a request. The value of the `Referer` header is the `tz1` address of the smart function (or account) that made the request.
 
 ```typescript
 async function handler(request: Request): Promise<Response> {
-    const referer = request.headers.get("Referer"); // "tz4..."
+    const referer = request.headers.get("Referer"); // "tz1..."
```

Comment:
do you know about any docs that say what exactly should be the difference between use of tz1, ... tz4 addresses?

Reply:
They're just different keypair algorithms -- tz1 is ed25519 and tz4 is BLS. Not aware of any docs apart from reading the code of the Tezos protocol